core/src/main/java/io/questdb/cairo/IndexBuilder.java

```diff
@@ -89,9 +89,9 @@ protected void rebuildColumn(
                 removeIndexFiles(columnName, ff, columnNameTxn);
                 TableUtils.dFile(path.trimTo(plen), columnName, columnNameTxn);
 
-                if (columnVersionReader.getColumnTopPartitionTimestamp(columnIndex) <= partitionTimestamp) {
+                final long columnTop = columnVersionReader.getColumnTop(partitionTimestamp, columnIndex);
```

Comment:
the log line "testing" is hard to comprehend when looking at the logs. Lets remove.

Instead, if column top = -1, we should log that we skip partition because column does not exist